qiskit/transpiler/target.py

```diff
@@ -590,6 +590,245 @@ def operation_names_for_qargs(self, qargs):
             raise KeyError(f"{qargs} not in target.")
         return res
 
+    def get_instruction_properties_by_class(self, operation_class, qargs, parameters=None):
```

Comment:
this is nitpicky but the other methods in this class don't start with `get_*` so for consistency I would drop `get_*` here too